instrumentation/google.golang.org/grpc/otelgrpc/config.go

```diff
@@ -131,3 +133,37 @@ func (o meterProviderOption) apply(c *config) {
 func WithMeterProvider(mp metric.MeterProvider) Option {
 	return meterProviderOption{mp: mp}
 }
+
+type event int
+
+// Different types of events that can be recorded, see WithMessageEvents.
+const (
+	ReceivedEvents event = iota
+	SentEvents
+)
+
+type messageEventsProviderOption struct {
+	events []event
+}
+
+func (o messageEventsProviderOption) apply(c *config) {
+	for _, e := range o.events {
```

Comment:
```suggestion
func (m messageEventsProviderOption) apply(c *config) {
	for _, e := range m.events {
```
The name of a method's receiver should be a one or two letter abbreviation of its type suffices.

Reply:
Fixed.